src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt

```diff
@@ -1,88 +0,0 @@
-package app.revanced.patches.youtube.misc.video.quality.patch
```

Comment:
This PR gets rid of the functionality to remember a quality. Instead it should only offer to change it manually in addition to offering the option to locking it. That way, you can either remember the last quality, change it manually in the settings and lock it optionally.

Reply:
It is handled in DefaultVideoQualityPatch